### PR TITLE
Add image registries annotation key

### PR DESCRIPTION
**Pull request by zhujian7**

### CODE REVIEW

**Comment by zhujian7** [comment]
cc @qiujian16 @zhiweiyin318

File: cluster/v1/types.go [hunk 1]
```diff
@@ -234,3 +234,18 @@ const (
 	// ClusterNameLabelKey is the key of a label to set ManagedCluster name.
 	ClusterNameLabelKey = "open-cluster-management.io/cluster-name"
 )
+
+const (
+	// ClusterImageRegistriesAnnotationKey is an annotation key on ManagedCluster to configure image override for addons
```
Comment (qiujian16) [inline cluster/v1/types.go:239]: I think we should also  mention addonDeploymentConfig will override this.

**Comment by zhujian7** [comment]
/unhold
cc @qiujian16 PTAL

File: cluster/v1/types.go (hunk 1 above)
Comment (zhujian7) [reply cluster/v1/types.go:239]: Fixed

**Pull request merged**